### PR TITLE
Move to Biome

### PR DESCRIPTION
**Pull request by kibertoad**
## Changes

Biome is faster and brings in way less dependencies.

## Checklist

- [X] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [X] I've updated the documentation, or no changes were necessary
- [X] I've updated the tests, or no changes were necessary

### CODE REVIEW

**Review by CarlosGamero — APPROVED** [review]

File: biome.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "extends": ["./node_modules/@lokalise/biome-config/configs/biome-base.jsonc"],
+  "linter": {
+    "rules": {
+      "performance": {
+        "noBarrelFile": "off",
```
[the remark this answers is not in the document]
Comment (kibertoad) [reply biome.json:7]: That rule is mostly useful for applications or larger libraries with plenty of optional imports. What would be the benefit of having indirect export structure in a library with just a few exports?

[the remark this answers is not in the document]
Comment (ondrejsevcik) [reply biome.json:7]: I would simply aim for consistency, and never use barrel files (no matter the size of the project). Or are there any scenarios where we still want to use them?

[the remark this answers is not in the document]
Comment (kibertoad) [reply biome.json:7]: note that if you don't have a barrel file in a library, you can no longer have imports like

import { Client, RequestError, EMPTY_SCHEMA } from 'my-library'

unless they all come from the same file. For me this also brings worse DX, or clutters code structure of the library.

So for libraries where tree-shaking is not important I'd still go with barrel files. maybe we need separate biome preset for them

[the remark this answers is not in the document]
Comment (ondrejsevcik) [reply biome.json:7]: Fair enough. We need to find the right balance with this.

**Pull request merged**